### PR TITLE
Add AWS logging to batch tests

### PR DESCRIPTION
**Pull request by jikawa-az**
Add AWS logging to the log batch unit tests so that the log files can verify the performance of readBatch and deleteStaleData functions.

Signed-off-by: Jesse Ikawa <jikawa@amazon.com>

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: cloudwatch_logs_common/src/utils/log_file_manager.cpp [hunk 1]
```diff
@@ -76,8 +76,8 @@ FileObject<LogCollection> LogFileManager::readBatch(
   }
 
   if(batch_size != log_data.size()){
-    AWS_LOG_INFO(__func__, "%d logs were not batched since the time"
-      " difference was > 24 hours. Will try again in a separate batch./n"
+    AWS_LOG_WARN(__func__, "%d logs were not batched since the time"
```
Comment (jaisontj) [inline cloudwatch_logs_common/src/utils/log_file_manager.cpp:79]: Generally it is good practice to have a PR do one thing; makes reverting easier. This looks like something that can be easily merged in another PR. But that is just my opinion, I don't know if its a big deal.

File: file_management/include/file_management/file_upload/file_manager.h [hunk 2]
```diff
@@ -236,7 +236,7 @@ class FileManager :
 
     if(logsDeleted > 0){
       AWS_LOG_INFO(__func__, "%d logs were deleted since the time"
-        " difference was > 14 days./n", logsDeleted
+        " difference was > 14 days.", logsDeleted
```
Comment (jaisontj) [inline file_management/include/file_management/file_upload/file_manager.h:239]: Same as my first comment.

File: cloudwatch_logs_common/src/utils/log_file_manager.cpp (hunk 1 above)
Comment (dabonnie) [reply cloudwatch_logs_common/src/utils/log_file_manager.cpp:79]: I think this is fine (small enough PR as well).

**Review by dabonnie — APPROVED** [review]
